adagrams/game.py

```diff
@@ -1,11 +1,212 @@
+
+import random
+LETTER_POOL = {
+    'A': 9,
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
+score_chart = {
+    1 : ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2 : ["D", "G"],
+    3 : ["B", "C", "M", "P"],
+    4 : ["F", "H", "V", "W", "Y"],
+    5 : ["K"],
+    8 : ["J", "X"],
+    10: ["Q", "Z"],
+}
+
+scores_chart = {
+    "A": 1, "C": 3, "B": 3, "E": 1, "D": 2, "G": 2, 
+    "F": 4, "I": 1, "H": 4, "K": 5, "J": 8, "M": 3, 
+    "L": 1, "O": 1, "N": 1, "Q": 10, "P": 3, "S": 1, 
+    "R": 1, "U": 1, "T": 1, "W": 4, "V": 4, "Y": 4, 
+    "X": 8, "Z": 10
+}
```

Comment:
Are these constant variables? If so, don't forget to follow the naming convention of uppercase characters! So in this case, `score_chart` should be `SCORE_CHART`.